docs/source/index.rst

```diff
@@ -97,6 +104,7 @@ TorchTune tutorials.
 
    examples/finetune_llm
    examples/lora_finetune
+   examples/qlora_finetune
```

Comment:
this is not in yet?

Reply:
Oops! Yeah some code from my other PR snuck in